README.md

```diff
@@ -9,63 +9,23 @@ It includes all the checks from the vcftools suite, and some more that involve l
 
 Please read the wiki for more details about checks already implemented.
 
-## Dependencies
-
-### Boost
-
-The dependencies are the Boost library core, and its submodules: Boost.filesystem, Boost.program_options, Boost.regex and Boost.system.
-If you are using Ubuntu, the required packages' names will be `libboost-dev`, `libboost-filesystem-dev`, `libboost-program-options-dev` and `libboost-regex-dev`.
-
-### ODB
-
-You will need to download the ODB compiler, the ODB common runtime library, and the SQLite database runtime library from [this page](http://codesynthesis.com/products/odb/download.xhtml).
-
-ODB requires SQLite3 to be installed. If you are using Ubuntu, the required packages' names will be `libsqlite3-0` and `libsqlite3-dev`.
-
-To install the ODB compiler, the easiest way is to download the `.deb` or `.rpm` packages, in order to be installed automatically with `dpkg`. Both the ODB runtime and SQLite database runtime libraries can be installed manually running `./configure && make && sudo make install`. This will install the libraries in `/usr/local/lib`.
-
-If you don't have root permissions, please run `./configure --prefix=/path/to/odb/libraries/folder` to specify which folder to install ODB in, then `make && make install`, without `sudo`.
-
 ## Build
 
-The build has been tested on the following compilers:
-* Clang 3.5 to 3.7
-* GCC 4.8 to 5.0
+The easiest way to build vcf-validator is using the Docker image provided with the source code. This will create a binary that can be run in any Linux machine.
 
-In order to create the build scripts, please run `cmake` with your preferred generator. For instance, `cmake -G "Unix Makefiles"` will create Makefiles, and to build the binaries, you will need to run `make`. If the ODB libraries were not found during the build, please run `sudo updatedb && sudo ldconfig`.
+1. Install and configure Docker following [their tutorial](https://docs.docker.com/engine/getstarted/).
+2. Create the Docker image:
+    1. Clone this Git repository
+    2. Move to the folder the code was downloaded to
+    3. Run `docker build -t my-name/vcf-validator docker/`. You can replace `my-name` with your name of choice.
```

Comment:
With actual example commands, you mean adding them explicitly to 1 and 2? I'm fine with that.

Explaining what `my-name` is would require a lot of text regarding what are Docker repositories... I can change the clarification to something like "You can replace `my-name` with any name that could identify you uniquely" and maybe add a link for those interested or in doubt. Otherwise suggestions are welcome.

The input VCF paths in the examples will obviously differ, but I hope a bioinformatician will be able to extrapolate their meaning :stuck_out_tongue: I don't understand the last question "Is this my-name equally unguessable?"